src/riak_object.erl

```diff
@@ -1189,23 +1189,25 @@ binary_version(<<131,_/binary>>) -> v0;
 binary_version(<<?MAGIC:8/integer, 1:8/integer, _/binary>>) -> v1.
 
 %% @doc Encode for nextgen_repl
--spec nextgenrepl_encode(repl_v1, riak_object(), boolean()) -> binary().
+-spec nextgenrepl_encode(
+    repl_v1,
+    riak_object()|
```

Comment:
Look at type here... object or reap tuple.

Reply:
The output of the function is a binary() though, or have I missed your point?  Formatting of this spec is quite confusing perhaps, I'll address this.